design-editor/src/brackets/brackets-editor.js

```diff
@@ -28,7 +28,9 @@ const beautyOptions = {
 	indent_char: ' ',
 	preserve_newlines: false,
 	unformatted: ['a', 'br', 'noscript', 'textarea', 'pre', 'code']
-};
+	},
+	brackets = utils.checkGlobalContext('brackets'),
+	isDemoVersion = utils.isDemoVersion(brackets);
```

Comment:
there is no need of passing arguments to `utils.isDemoVersion`. It now does not require them anymore.